packages/inquirer/examples/input.js

```diff
@@ -29,12 +23,11 @@ const questions = [
     name: 'fav_color',
     message: "What's your favorite color",
     transformer(color, answers, flags) {
-      const text = chalk.hex(isHex(color) ? color : 'fff')(color);
```

Comment:
this was a fun example but picocolors doesn't support hex colours, so it doesn't make much sense anymore

i opted to just remove it rather than keep a dep just for an example. i suppose if we're not running these examples locally, we could leave it as-is, knowing it may not run in the repo itself

Reply:
I reckon this isn’t a real deal breaker, but an alternative might be https://github.com/webdiscus/ansis, it supports 16 colours rather than 8 and is just a little bigger in size. Not done deep analysis on it though.

Reply:
keep in mind this is only for the example, too. we could actually leave the example as using chalk, or whatever package really

inquirer itself doesn't make use of hex colours so should be fine on picocolors

Reply:
Yeah just thought I’d shout it out while taking a look :).